pumps/sql_test.go

```diff
@@ -363,10 +383,11 @@ func TestSQLWriteUptimeDataAggregations(t *testing.T) {
 }
 
 func TestDecodeRequestAndDecodeResponseSQL(t *testing.T) {
+	skipTestIfNoPostgres(t)
 	newPump := &SQLPump{}
 	cfg := make(map[string]interface{})
-	cfg["type"] = "sqlite"
-	cfg["connection_string"] = ""
+	cfg["type"] = "postgres"
+	cfg["connection_string"] = getTestPostgresConnectionString()
```

Comment:
I see this repeated many times, what about creating a kind of constructor for this? something like:

```
NewSQLConfig(shard bool)map[string]interface{}{
        cfg := make(map[string]interface{})
	cfg["type"] = "postgres"
	cfg["connection_string"] = getTestPostgresConnectionString()
	cfg["table_sharding"] = shard
	return cfg
}
```